auctions/english-auction/src/events.sw

```diff
@@ -0,0 +1,28 @@
+library events;
+
+dep data_structures;
+
+use data_structures::{Asset, Auction};
+use std::identity::Identity;
+
+pub struct AuctionCancelEvent {
+    auction: Auction,
+    auction_id: u64,
+}
+
+pub struct AuctionStartEvent {
```

Comment:
I think "created" makes more sense than start since you don't create it and then wait, correct? I assume you create it and start it instantly

Reply:
`CreateAuctionEvent`?